scripts/Reader.py.experimental

```diff
@@ -201,8 +201,11 @@ class Reader(object):
         else:
             with open(path + '/deviceName.txt', 'r') as f:
                 device_name = f.read().rstrip().split(';', 1)[0]
-            with open(path + '/../settings/Mode_Legacy', 'r') as f:
-                mode_legacy = f.read().rstrip().split(';', 1)[0] == 'ON'
+            try:
+                with open(path + '/../settings/Mode_Legacy', 'r') as f:
+                    mode_legacy = f.read().rstrip().split(';', 1)[0] == 'ON'
+            except FileNotFoundError:
```

Comment:
i would prefer checking file existence with `if os.path.isfile(...)` rather can producing exceptions. The exception handling always has some overhead.

Reply:
Done, cf. 08365b1eda851b55643b0edd7f09015eae9ad4b2.